deps/src/main/java/com/microsoft/azure/sdk/iot/deps/util/Base64.java

```diff
@@ -0,0 +1,324 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package com.microsoft.azure.sdk.iot.deps.util;
+
+public final class Base64
```

Comment:
I got rid of this code yesterday since we should be using apache commons for our base64 encoding/decoding. Don't re-add this class back into our codebase

Reply:
I merged the changes from master, weird it's showing up. Let me check what happened.